.circleci/config.yml

```diff
@@ -256,7 +256,7 @@ jobs:
       - attach_workspace: *attach_options
       - run:
           working_directory: docs
-          command: ./generate_documentation.sh
+          command: yarn workspace raiden-ts run docs && cp -r ./raiden-ts/docs ./raiden-dapp/dist/
```

Comment:
Please consider to make this a multiline command instead of chaining it. If you need the conditional chain use `command: >`, else just `command: |`. It think it reads better and in this file in specific we should try to be as easy on the eyes as possible.
But this is a tiny nitpick....

Reply:
:heavy_check_mark: